docs/operating-scylla/index.rst

```diff
@@ -58,7 +58,7 @@ ScyllaDB for Administrators
   :id: "getting-started"
   :class: my-panel
    
-  * :doc:`Learn To Use Scylla </using-scylla/learn>` - Scylla University offers many courses.  Pick a course and get started on your path to becoming a Scylla expert.
+  * `Learn To Use Scylla <https://docs.scylladb.com/stable/get-started/learn-resources/index.html>`_ - Scylla University offers many courses.  Pick a course and get started on your path to becoming a Scylla expert.
```

Comment:
Is this page maintained outside docs/?

Reply:
This links seems to be out of scope for **operating scylla** and for the PR subject.

Reply:
The section is "Operating Scylla" so IMHO there is no need for a link to a dev getting started.

Reply:
> One thing I can do is remove this link from /operating-scylla/, but I'd rather not. The Learn page includes information about example projects for devs and University, but also links to our pages with events, webinars, etc., which are super useful to administrators, too.

IMHO we can safly remove it.
Users can easly find /using-scylla no need to link to it from every page.